zookeeper-server/src/main/java/org/apache/zookeeper/server/ZooKeeperServer.java

```diff
@@ -530,6 +530,10 @@ public void takeSnapshot(boolean syncSnap) {
         ServerMetrics.getMetrics().SNAPSHOT_TIME.add(elapsed);
     }
 
+    public boolean shouldForceWriteInitialSnapshotDuring34Migration() {
```

Comment:
there is no need to add 'During34Migration', we can just leave a javadoc

Reply:
Ok, will fix. How about `shouldForceWriteInitialSnapshotAfterLeaderElection`?